components/FlashcardForm.js

```diff
@@ -56,6 +56,8 @@ export default function FlashcardForm({
   onClose,
   collections,
 }) {
+  const { data: session } = useSession();
+  const userId = session.user.id;
```

Comment:
check if the session exists before accessing the id
```suggestion
  const userId = session?.user.id;
```

Reply:
suggestion adopted. I forget this method for checking the exists.